src/Volume.ts

```diff
@@ -244,19 +246,32 @@ export default class Volume {
     let noReload =
       this.loadSpec.time === this.loadSpecRequired.time &&
       this.loadSpec.subregion.containsBox(this.loadSpecRequired.subregion) &&
-      this.loadSpecRequired.channels.every((channel) => this.loadSpec.channels.includes(channel));
+      this.loadSpecRequired.channels.every((channel) => this.loadSpec.channels.includes(channel)) &&
+      this.loadSpecRequired.scaleLevelBias === this.loadSpec.scaleLevelBias;
 
     // An update to `subregion` should trigger a reload when the new subregion is not contained in the old one
     // OR when the new subregion is smaller than the old one by enough that we can load a higher scale level.
-    if (noReload && !this.loadSpec.subregion.equals(this.loadSpecRequired.subregion)) {
+    if (
+      noReload &&
+      (!this.loadSpec.subregion.equals(this.loadSpecRequired.subregion) ||
+        this.loadSpecRequired.maxAtlasEdge !== this.loadSpec.maxAtlasEdge ||
+        this.loadSpecRequired.multiscaleLevel !== this.loadSpec.multiscaleLevel)
+    ) {
```

Comment:
when there are still many conditionals I prefer to pull those out into a function whose name tells you what you're ultimately checking for so this if statement is easier to read